server/embed/serve.go

```diff
@@ -93,7 +94,14 @@ func (sctx *serveCtx) serve(
 	errHandler func(error),
 	gopts ...grpc.ServerOption) (err error) {
 	logger := defaultLog.New(io.Discard, "etcdhttp", 0)
-	<-s.ReadyNotify()
+
+	// When the quorum isn't satisfied, then etcd server will be blocked here
+	// forever, accordingly it can't even serve serializable read request.
```

Comment:
This comment is doesn't describe current state,  but the state from before making the line change.

Reply:
Right, thanks. Updated.